spec/system/passenger_management_spec.rb

```diff
@@ -148,56 +151,52 @@
     end
   end
 
-  context 'as a dispatcher' do
-    before do
-      @user = create(:user)
-      @passenger = create(:passenger, name: 'Foo Bar')
-      when_current_user_is(@user)
-    end
+  context 'when the user is a dispatcher' do
+    before { when_current_user_is :anyone }
```

Comment:
Here's another rabbit hole, I guess users are either dispatchers or admin. fun stuff, but it's fine for now

Reply:
It being a dispatch-only tool, the base type of `User` is a dispatcher. This app is a little weird in that you can log in as a `Passenger`, too. The helper method also accepts `:anybody`, I could add `:dispatcher` as another synonym, but I wonder if that would misleadingly imply some sort of special permissions?

Reply:
That's what I wasn't sure about, whether or not passengers could log in. Unless it's something you want to fix particularly bad I'm okay with this staying as is based on [this tidbit](https://github.com/umts/st-pax/blob/19bbdae20664f89fb61abee5508c3b2dbd26e762/app/models/user.rb#L11)